src/odemis/driver/spectrapro.py

```diff
@@ -353,26 +354,26 @@ def _sendQuery(self, com, timeout=1):
                     raise
 
         # read response until timeout or known end of response
-        response = ""
+        response = b""
         timeend = time.time() + timeout
         while ((time.time() <= timeend) and
-               not (response.endswith(" ok\r\n") or response.endswith("? \r\n"))):
+               not (response.endswith(b" ok\r\n") or response.endswith(b"? \r\n"))):
             self._serial.timeout = max(0.1, timeend - time.time())
             char = self._serial.read()
             if not char: # timeout
                 break
             response += char
 
-        logging.debug("Received: %s", response.encode('string_escape'))
-        if response.endswith(" ok\r\n"):
-            return response[:-5]
```

Comment:
I changed the interface of the sendQuery function compared to the other drivers, here I return unicode. I think that I don't need any changes in the functions that call sendQuery if I do it like this. Do you think it's a good idea?

Reply:
Alternatively, we could decode in python3 and do nothing in python2, so we always return str. I think that might be even better.

Reply:
I'm fine with the simple `decode("latin1")`.
Eventually, we will only run on Python 3, so the more the code looks like the final code, the better.